src/golang/lib/job/errors.go

```diff
@@ -10,22 +10,22 @@ type JobErrorCode int
 
 const (
 	// System indicates an unexpected system issue that we cannot recover from.
-	System JobErrorCode = 0
+	System JobErrorCode = iota
 
 	// User error code indicates that the issue was the user's fault, and to surface that message
 	// to the user.
-	User = 1
+	User = iota
```

Comment:
Wait you don't need to set the other ones to iota right, it goes:
```
System JobErrorCode = iota
User
JobMissing
Noop
```

Reply:
got it thx!